Gemfile

```diff
@@ -4,6 +4,7 @@ source 'http://rubygems.org'
 
 gemspec
 
-# gem 'killbill-client', :git => 'https://github.com/killbill/killbill-client-ruby.git', :branch => 'work-for-release-0.21.x'
+gem 'killbill-client', :git => 'https://github.com/killbill/killbill-client-ruby.git', :branch => 'master'
```

Comment:
I think the PR should go against `work-for-release-0.21.x` (that way, you can keep `work-for-release-0.21.x` here).

Reply:
Also, Rubocop is also complaining (https://stackoverflow.com/questions/44005410/need-to-refactor-to-the-new-ruby-1-9-hash-syntax):

```
Gemfile:7:24: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
gem 'killbill-client', :git => 'https://github.com/killbill/killbill-client-ruby.git', :branch => 'master'
                       ^^^^^^^
Gemfile:7:88: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
gem 'killbill-client', :git => 'https://github.com/killbill/killbill-client-ruby.git', :branch => 'master'
                                                                                       ^^^^^^^^^^
```

Reply:
Not sure I understand the following:
> I think the PR should go against work-for-release-0.21.x

This references `killbill-client-ruby`. This repo only has the `master` branch. There is no `work-for-release-0.21.x` branch here.

I've fixed the Rubocop issues.

Reply:
Oh, indeed, we are missing the `work-for-release-0.23.x` branch in that repo, I've just created it. I'll fix the `Gemfile`.